demos/top-app-bar.scss

```diff
@@ -55,3 +55,21 @@
 input[type="checkbox"]:disabled + label {
   opacity: .4;
 }
+
+
+.demo-col {
+  display: flex;
+  flex: 1 1 auto;
+  flex-direction: column;
+  margin: 0 20px 20px 20px;
+
+  & > span {
```

Comment:
We should really avoid element selectors... Could we use something like `<h4 class="mdc-typography--subheading2 mdc-typography--adjust-margin` for these elements instead? (And maybe use mdc-typography--title on the Demo Controls heading?)